Golf-Drill-Challenge-App/Golf-App
# Gehrkej/Reset season

### Changes
---
This PR will add the functionality for clearing the leaderboards which will be the first aspect of resetting a season. A future feature could include a screen to remove users that have left the team or graduated. 

**Related issue:** #189 

### How to test:
---

1. Login to an "owner" account
2. Look at a leaderboard that currently has attempts
3. Go to the team page
4. Click the icon in the top right
5. Select "reset season"
6. Confirm action
7. Check the same leaderboard to see if it has been wiped

### Screenshots
---
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/6e585bf6-632e-4820-b6d2-392f7a49f608)\
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/7b2cd121-0ab3-4493-a252-a57e3ee3fc3f)
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/e1bc464a-d5e7-46fa-8088-e87e66105e3c)
